.gitignore

```diff
@@ -49,3 +50,8 @@ compile_commands.json
 .vscode/*
 !.vscode/extensions.json
 .DS_Store
+
+# Build binaries
+/build/addons/numdot/**/*
+!/build/addons/numdot/**/
+!/build/addons/numdot/**/*.plist
```

Comment:
I assume this change is not meant to be included in this PR?

Reply:
Oh, right, that should be `/build/addons/gdexample/`. Fixing